File: Resueltos_por_competidor/German_Scioville/MissingNumber.java
/*
 * Autor: Scioville
 * Problema: Missing Number (https://cses.fi/problemset/task/1083)
 * Juez online: CSES
 * Veredicto: accepted  
 * Jesus Christ is in Heaven Now
 * */
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

public class MissingNumber {


	public static void main(String[] args) throws IOException {
		
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		int n = Integer.parseInt(br.readLine());
		
		String [] lista = br.readLine().split(" ");
		
		int[] numeros = new int[lista.length];
		for(int i = 0;i<n-1;i++) {
			numeros[i] = Integer.parseInt(lista[i]);
		}
		
		Arrays.sort(numeros);
		
		int aux=1;
		for(int i = 0;i<n-1;i++) {
			if(aux != numeros[i]) {
				System.out.println(aux);
				return;
			}
			
			aux++;
		}
		
		System.out.println(aux);
		
	}
}

File: Resueltos_por_competidor/Juan_Martinez/MissingNumber.cpp
/*
 * Autor: Juan Martinez
 * Problema: Missing Number
 * Juez online: Cses
 * Veredicto: Accepted
 * Url: https://cses.fi/problemset/task/1083/
 */
#include <bits/stdc++.h>
using namespace std;

int main() {
	int n, temp; cin>>n;
    set<int> numbers;
    for(int i = 0; i < (n-1); i++) cin>>temp, numbers.insert(temp);
    for(int i = 1; i <= n; i++)  {
        if(numbers.count(i) == 0) {
            cout<<i<<endl;
            break;
        }
    }
}
